### ISSUE TITLE
Smokehouse executable fails with ERR_MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by tlgs** [GoogleChrome/lighthouse#16647]
### FAQ

- [x] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [x] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).
- [x] Yes, my issue is not answered by [other FAQs](https://github.com/GoogleChrome/lighthouse#faq).

### URL

https://example.com

### What happened?

1. Install the lighthouse Node CLI: `npm install -g lighthouse`
2. Invoke the smokehouse executable: `smokehouse`

This results in `Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/node_modules/lighthouse/cli/test/smokehouse/test-definitions/a11y.js' imported from /usr/lib/node_modules/lighthouse/cli/test/smokehouse/core-tests.js`

### What did you expect?

Probably the tool informing me I'm missing some keyword arguments. :)

### What have you tried?

I've tried looking for other examples of how `smokehouse` works and how it's intended to be used. I have found no information outside of [cli/test/smokehouse/readme](https://github.com/GoogleChrome/lighthouse/blob/main/cli/test/smokehouse/readme.md) and unfortunately my knowledge as a surface-level lighthouse user isn't enough.

### How were you running Lighthouse?

CLI

### Lighthouse Version

12.8.1

### Chrome Version

chromium 139.0.7258.66-1

### Node Version

24.6.0

### OS

Linux

### Relevant log output

```sh
node:internal/modules/esm/resolve:274
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/lib/node_modules/lighthouse/cli/test/smokehouse/test-definitions/a11y.js' imported from /usr/lib/node_modules/lighthouse/cli/test/smokehouse/core-tests.js
    at finalizeResolution (node:internal/modules/esm/resolve:274:11)
    at moduleResolve (node:internal/modules/esm/resolve:864:10)
    at defaultResolve (node:internal/modules/esm/resolve:990:11)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:735:20)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:712:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:310:38)
    at #link (node:internal/modules/esm/module_job:208:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///usr/lib/node_modules/lighthouse/cli/test/smokehouse/test-definitions/a11y.js'
}
```

**Comment by connorjclark** [comment]
Just checking, but are you developing a plugin? Outside of testing the Lighthouse end-to-end in this repo, that's the only use for smokehouse.

The tests that run by default for core development are in `cli/test/smokehouse/core-tests.js`. The tool loads that by default. When run from the lighthouse npm package w/o setting `--tests-path`, that would error since we [don't ship the core tests to npm](https://github.com/GoogleChrome/lighthouse/blob/ddcf7d330af839f5bb2bccb46e2a90ecb69365c0/.npmignore#L29) (the fact that `core-tests.js` is shipped is an oversight).

And yeah, we should emit a specific error for the case where the core tests can't load and point plugin developers to the page for smokehouse documentation.

Plugin developers can run their own smoke tests like this (shown at the bottom of the smokehouse doc):

```
npx --no-install smokehouse  --tests-path plugin-tests.js
```

The format should be the same as seen in https://github.com/GoogleChrome/lighthouse/blob/main/cli/test/smokehouse/core-tests.js . Each module it exports conforms to this [shape](https://github.com/GoogleChrome/lighthouse/blob/ddcf7d330af839f5bb2bccb46e2a90ecb69365c0/types/internal/smokehouse.d.ts#L36).

**Comment by tlgs** [comment]
Thanks for the quick reply! No, I am the [lighthouse package](https://aur.archlinux.org/packages/lighthouse) maintainer for the Arch User Repository. This is meant for the CLI application and not the library.
Since both `/usr/bin/smokehouse` and `usr/bin/chrome-debug` are symlinked when installing the lighthouse Node CLI, I figured I'd ask. 

I'm thinking I should exclude both of these and only keep the `lighthouse` install. What do you think?

**Comment by Spixmaster** [comment]
@tlgs Keep it how it is installed by `npm`, i. e. how it currently is.

This has the advantage that the same package behaves similarly across distributions and utlimately the developers have the decision what files to install.
